### PR TITLE
Support Scala.js 1.x where x > 0

### PR DESCRIPTION
**Pull request by lolgab**
- Add test for Scala.js 1.x (latest) leaving the 1.0 test to ensure
  we don't break backward compatibility with older Scala.js versions

### CODE REVIEW

**Review by tgodzik — APPROVED** [review]
LGTM from my side, thanks for the contribution! @sjrd would it be possible for you to confirm?

File: project/Dependencies.scala [hunk 1]
```diff
@@ -46,7 +46,9 @@ object Dependencies {
   val scalaNative03Version = "0.3.9"
   val scalaNative04Version = "0.4.0-M2"
   val scalaJs06Version = "0.6.32"
-  val scalaJs10Version = "1.0.1"
+  val scalaJs1Version = "1.3.1"
+  val scalaJsEnvsVersion = "1.1.1"
+  val scalaJsEnvJsdomNodejsVersion = "1.1.1"
```
Comment (sjrd) [inline project/Dependencies.scala:51]: This is unused. (And if were used, it should be 1.1.0.)

**Review by sjrd — APPROVED** [review]

File: project/Dependencies.scala (hunk 1 above)
Comment (tgodzik) [reply project/Dependencies.scala:51]: @lolgab could you remove `scalaJsEnvJsdomNodejsVersion` ?

Comment (lolgab) [reply project/Dependencies.scala]: Ops, I was adding it and then I went for a string literal, since there was only a single usage.
Removed ✅

Comment (tgodzik) [reply project/Dependencies.scala]: Awesome! Thanks!

**Pull request merged**